src/main/java/com/thomashan/coup/game/GameValidator.java

```diff
@@ -0,0 +1,19 @@
+package com.thomashan.coup.game;
+
+public class GameValidator {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A class which only has private constructors should be final](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4561385)